### PR TITLE
[FIX] Double scroll on 'keyboard shortcuts' menu in sidepanel

### LINKED ISSUE #7901: Double scroll on 'keyboard shortcuts' menu in sidepane.
### Description:

screenshot from 2017-08-25 19-33-57

### Server Setup Information:

demo server. 0.59.0.rc-1

### Steps to Reproduce:

1. Open `keyboard shortcuts` in sidepane.

### Expected behavior:

single scroll on right side.

### Actual behavior:

Double scroll on right.

### PR DESCRIPTION
**Pull request by aditya19496**
@RocketChat/core 

Closes #7901

### CODE REVIEW

File: packages/rocketchat-ui-flextab/client/tabs/keyboardShortcuts.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template name="keyboardShortcuts">
-    <div class="content">
+    <!-- <div class="content"> -->
```
Comment (MartinSchoeler) [inline packages/rocketchat-ui-flextab/client/tabs/keyboardShortcuts.html]: Please remove the tag instead of commenting

Comment (aditya19496) [reply packages/rocketchat-ui-flextab/client/tabs/keyboardShortcuts.html]: Sure, will do the same.

**Comment by jszaszvari** [comment]
@aditya19496  @rodrigok  @karlprieb 

Not sure this is actually fixed. 

In 0.59.x it looks like this with the double scroll
image


But now in 60.2 it looks like this:
image

**Comment by cyclops24** [comment]
@karlprieb @rodrigok I also confirmed that it's break in version 60 release.

**Pull request merged**